### PR TITLE
ta: cleanup trace messages in test TAs

### PR DESCRIPTION
**Pull request by etienne-lms**
This is a follow up of cleanup changes initiated in https://github.com/OP-TEE/optee_test/pull/708 and https://github.com/OP-TEE/optee_test/pull/709 to have TA trace messages implementation more consistent with OP-TEE coding style and trace resources.

### CODE REVIEW

**Review by jbech-linaro — COMMENTED** [review]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

File: ta/os_test/include/tb_macros.h [hunk 1]
```diff
@@ -8,43 +8,43 @@
...
+	MSG("\n*********** TESTBENCH ***********" \
+	    "\n***         PASSED:  <<< %s >>>"   \
+	    "\n*********************************", str)
+
+#define TB_INFO(str) MSG("*** INFO : %s \n", (str))
+
+#define HALT                                                           \
+	{                                                              \
+		EMSG("\n*** FAILED ***"                                \
+		       "\nTestbench halted at line %d in function %s"; \
```
Comment (vincent-mailhol) [inline ta/os_test/include/tb_macros.h:26]:
```suggestion
		EMSG("\n*** FAILED ***"                                \
		     "\nTestbench halted at line %d in function %s"; \
```
Fix the alignment.

**Review by vincent-mailhol — COMMENTED** [review]
Global comments: in the commit description, the best practise is to use the imperative tone. This means no third person "s" in the verbs. e.g. s/Uses/Use/g, s/Adds/Add/g, s/Removes/Remove/g and so on.


**Reference:** https://www.kernel.org/doc/html/v4.10/process/submitting-patches.html#describe-your-changes

> Describe your changes in imperative mood, e.g. “make xyzzy do frotz” instead of “[This patch] makes xyzzy do frotz” or “[I] changed xyzzy to do frotz”, as if you are giving orders to the codebase to change its behaviour.

**Comment by etienne-lms** [comment]
Thanks @vincent-mailhol for the review.

> Global comments: in the commit description, the best practise is to use the imperative tone. (...)

@jforissier, @jbech-linaro, is this the preferred phrasing in OP-TEE repos commits?

**Comment by jforissier** [comment]
> Thanks @vincent-mailhol for the review.
> 
> > Global comments: in the commit description, the best practise is to use the imperative tone. (...)
> 
> @jforissier, @jbech-linaro, is this the preferred phrasing in OP-TEE repos commits?

Yes.

**Comment by etienne-lms** [comment]
@vincent-mailhol, maybe you want to post a review tag for this series?

**Comment by vincent-mailhol** [comment]
> @vincent-mailhol, maybe you want to post a review tag for this series?

I am fine with the contents. But my main comment is on the commit description and I do not see changes. This is why I have not given my review tag yet.

**Comment by etienne-lms** [comment]
Ok, thanks for the feedback . I'll squash the fixup commits right away.

**Comment by etienne-lms** [comment]
Done

**Comment by vincent-mailhol** [comment]
For the series:

`Reviewed-by: Vincent Mailhol <mailhol.vincent@wanadoo.fr>`

Thank you.

**Comment by jbech-linaro** [comment]
> @jbech-linaro, could you infirm or confirm your review tag?

indeed!

**Comment by jbech-linaro** [comment]
PR probably needs a rebase as well, since we see the pkcs11 1007 error here as well.

**Pull request merged**